examples/chart/teleport-cluster/values.yaml

```
@@ -263,3 +263,7 @@ priorityClassName: ""
 # Tolerations for pod assignment
 # https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
 tolerations: []
+
+# Timeouts for the readiness and liveness probes
+# https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+probeTimeout: 1
```

Suggestion: mention time unit, bump defaults a bit.

```suggestion
probeTimeoutSeconds: 5
```